ts/components/screens/BaseHeader.tsx

```diff
@@ -33,18 +33,18 @@ export class BaseHeader extends React.PureComponent<Props> {
     return (
       <AppHeader primary={this.props.primary} noLeft={goBack !== true}>
         {goBack && (
-          <Left>
+          <Left style={{ flex: 0 }}>
```

Comment:
The change must be done in the theme of the native-base Header component.

Reply:
https://github.com/teamdigitale/io-app/blob/master/ts/theme/components/Header.ts